code/datums/tutorial/marine/hospital_corpsman_sandbox.dm

```diff
@@ -0,0 +1,682 @@
+
+	addtimer(CALLBACK(src, PROC_REF(spawn_agents)), 2 SECONDS)
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/end_supply_phase()
+
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/structure/machinery/door/airlock/multi_tile/almayer/medidoor, prep_door)
+	var/turf/boundry = get_turf(loc_from_corner(4, 1))
+	if(tutorial_mob.x <= boundry.x)
+		message_to_player("Please exit the preperations room before progressing into the next round!")
+		return
+	prep_door.close(TRUE)
+	prep_door.lock(TRUE)
+	stage = TUTORIAL_HM_PHASE_MAIN
+	remove_action(tutorial_mob, /datum/action/hm_tutorial/sandbox/ready_up)
+	handle_round_progression()
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/begin_supply_phase()
+
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/structure/machinery/door/airlock/multi_tile/almayer/medidoor, prep_door)
+	prep_door.unlock(TRUE)
+	prep_door.open()
+	stage = TUTORIAL_HM_PHASE_MAIN // just in case it wasnt already
+
+	message_to_player("Phew! We have entered a resupply phase of the tutorial!")
+	message_to_player("Use this rare opportunity to refill, restock, and resupply yourself for future rounds.")
+	message_to_player("Remember, on the field, immediate resupply will not always be possible! You won't know for certain when your next chance will arrive, so stock up while you can!")
+	message_to_player("When you are ready, leave the supply room, then click the 'Ready Up' action on the top left of your screen to begin your next round.")
+
+	give_action(tutorial_mob, /datum/action/hm_tutorial/sandbox/ready_up, null, null, src)
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/spawn_agents()
+	SIGNAL_HANDLER
+
+	for(var/i in 1 to (round(rand(min_survival_agents, max_survival_agents))))
+		var/mob/living/carbon/human/realistic_dummy/active_agent = new(agent_spawn_location)
+		arm_equipment(active_agent, /datum/equipment_preset/uscm/tutorial_rifleman)
+		var/turf/dropoff_point = loc_from_corner(round(rand(6, 8), 1), round(rand(1, 3)))	// Picks a random turf to move the NPC to
+		agents[active_agent] = dropoff_point
+		active_agent.a_intent = INTENT_DISARM
+		simulate_condition(active_agent)
+		var/obj/item/clothing/suit/storage/marine/medium/armor = active_agent.get_item_by_slot(WEAR_JACKET)
+		RegisterSignal(armor, COMSIG_ITEM_UNEQUIPPED, PROC_REF(item_cleanup))
+
+	addtimer(CALLBACK(src, PROC_REF(eval_agent_status)), 3 SECONDS)	// Gives time for NPCs to pass out or die, if their condition is severe enough
+	if((survival_difficulty >= TUTORIAL_HM_INJURY_SEVERITY_FATAL) && (rand() <= 0.75))	// If above difficulty FATAL, starts a random timer to spawn a booboo agent
+		booboo_timer = addtimer(CALLBACK(src, PROC_REF(eval_booboo_agent)), (rand(15,25)) SECONDS, TIMER_STOPPABLE)
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/simulate_condition(mob/living/carbon/human/target)
+	SIGNAL_HANDLER
+
+	// Simulates patient NPC injuries
+
+	var/damage_amount_split = ((round(rand(1, 100))) / 100)	// How damage should be split between brute and burn
```

Comment:
when given a lower and upper bound rand() returns an integer so round() is not needed https://www.byond.com/docs/ref/#/proc/rand
I would probably just redo this line.